### PR TITLE
bug: fixed image size not counting config and manifest sizes

### PR DESCRIPTION
**Pull request by aokirisaki**
Signed-off-by: Ana-Roberta Lisca <ana.kagome@yahoo.com>

**What type of PR is this?**

**Which issue does this PR fix**:
#927 

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by rchincha** [comment]
* zotregistry.io/zot/pkg/cli/image_cmd_test.go 
  Line 962:
  Expected 'IMAGE NAME TAG DIGEST SIGNED SIZE repo7 test:1.0 883fc0c5 false 15B repo7 test:2.0 883fc0c5 false 15B' to contain substring 'repo7 test:2.0 883fc0c5 false 492B' (but it didn't)!

  * zotregistry.io/zot/pkg/cli/image_cmd_test.go 
  Line 1018:
  Expected 'IMAGE NAME TAG DIGEST SIGNED SIZE repo7 test:1.0 883fc0c5 false 15B repo7 test:2.0 883fc0c5 false 15B' to contain substring 'repo7 test:2.0 883fc0c5 false 492B' (but it didn't)!


274 total assertions

--- FAIL: TestServerResponseGQL (0.98s)
FAIL
coverage: 27.8% of statements in ./...
FAIL	zotregistry.io/zot/pkg/cli	138.188s

File: pkg/cli/client.go [hunk 1]
```diff
@@ -290,6 +290,15 @@ func (p *requestsPool) doJob(ctx context.Context, job *manifestJob) {
 		)
 	}
 
+	size += uint64(job.manifestResp.Config.Size)
+
+	manifestSize, err := strconv.Atoi(header.Get("content-length"))
+	if err != nil {
+		panic(err)
```
Comment (andaaron) [inline pkg/cli/client.go:297]: Use `p.outputCh <- stringResult{"", err}` same as in the other cases.

File: pkg/extensions/search/resolver.go [hunk 2]
```diff
@@ -600,6 +600,9 @@ func BuildImageInfo(repo string, tag string, manifestDigest godigest.Digest,
 			})
 		}
 
+		// manifestSize := olu.GetImageManifestSize(repo, manifest.Digest)
```
Comment (andaaron) [inline pkg/extensions/search/resolver.go:603]: Please remove these comments.

**Review by andaaron — DISMISSED** [review]

**Review by andaaron — APPROVED** [review]

File: pkg/cli/client.go [hunk 3]
```diff
@@ -290,6 +290,15 @@ func (p *requestsPool) doJob(ctx context.Context, job *manifestJob) {
 		)
 	}
 
+	size += uint64(job.manifestResp.Config.Size)
+
+	manifestSize, err := strconv.Atoi(header.Get("content-length"))
```
Comment (rchincha) [inline pkg/cli/client.go:295]: "Content-Length"
This is the right mnemonic to read this

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**